app/AppKernel.php

```diff
@@ -25,6 +25,7 @@
  */
 
 use PrestaShop\PrestaShop\Adapter\Module\Repository\ModuleRepository;
+use PrestaShop\PrestaShop\Adapter\SymfonyKernel;
```

Comment:
why this namespace was missing?

Reply:
That comes from a class that was added for testing and was deleted, I forgot to remove that line.